class/defaults.yml

```diff
@@ -151,6 +151,7 @@ parameters:
       vshn:
         enabled: false
         secretNamespace: ${crossplane:namespace}
+        mailgun_smtp_password: "?{vaultkv:${cluster:tenant}/${cluster:name}/appcat/mailgun_smpt_password}"
```

Comment:
Is there a reason for not using camel case here?

Reply:
Fixed